data/postsrsd.conf.in

```diff
@@ -152,22 +156,30 @@ hash-minimum = 4
 always-rewrite = off
 
 # Execute PostSRSd as unprivileged user
-# If you set this highly recommended option, PostSRSd will drop root
-# privileges and switch to the configured user before it enters the main loop
-# to handle untrusted input.
+# Drop root privileges and run as this user before entering the main loop and
+# handling untrusted input. To prevent PostSRSd from changing users, set this to
+# the empty string.
 #
-# Example:
+# Default if unset:
+#     unprivileged-user = "@POSTSRSD_USER@"
+#
+# Examples:
 #     unprivileged-user = "nobody"
+#     unprivileged-user = "" # Don't drop root privileges.
 #
 unprivileged-user = "@POSTSRSD_USER@"
 
 # Execute PostSRSd in chroot jail
-# If you set this highly recommended option, PostSRSd will jail itself
-# in the given directory, which adds an additional layer of protection
-# against the exploitation of security bugs in PostSRSd.
+# PostSRSd will jail itself in the given directory, which adds an additional
+# layer of protection against the exploitation of security bugs in PostSRSd. To
+# prevent PostSRSd from chrooting, set this to the empty string.
 #
-# Example:
+# Default if unset:
+#     chroot-dir = "@POSTSRSD_CHROOTDIR@"
+#
+# Examples:
 #     chroot-dir = "@CMAKE_INSTALL_FULL_LOCALSTATEDIR@/lib/@PROJECT_NAME@"
+#     chroot-dir = "" # Don't chroot.
```

Comment:
This looks very awkward to me. Maybe we can explain at the beginning of the file how to set an empty string, i.e. `= ""`. The description already documents what an empty value means, no need to repeat it as a comment.

Reply:
Done.